astropy/stats/lombscargle/statistics.py

```diff
@@ -0,0 +1,492 @@
+"""
+Utilities for computing periodogram statistics.
```

Comment:
maybe point out this is meant to be an internal module?  Just so no later maintainer thinks "hey, I should import this into the public namespace"